source/tsconfig-json.d.ts

```diff
@@ -990,6 +934,71 @@ declare namespace TsConfigJson {
 		explainFiles?: boolean;
 	}
 
+	namespace WatchOptions {
+		export type WatchFileKind =
```

Comment:
FYI I chose these names to be consistent with:
https://github.com/microsoft/TypeScript/blob/f9e6ba38f1cf38f7e5956396f548fce37934f9cc/lib/protocol.d.ts#L1121